torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -910,15 +940,19 @@ def generate_detector_report(self, model: GraphModule) -> Tuple[str, Dict[str, A
 
             mod_info: Dict[str, Any] = input_weight_equalization_info[module_fqn]
 
-            # look at each individual channel and add a suggestion
-            for index, channel_suggested in enumerate(mod_info[self.RECOMMENDED_KEY]):
-                if channel_suggested:
-                    channel_str = channel_suggestion_str.format(index, use_str, input_weight_benefit_str)
-                    input_weight_string += channel_str
-                else:
-                    non_benefit_str = input_weight_non_benefit_str.format(input_weight_non_benefit_reasoning)
-                    channel_str = channel_suggestion_str.format(index, no_use_str, non_benefit_str)
-                    input_weight_string += channel_str
+            # gather info on how many channels would benefit from input weight and
```

Comment:
nit: would be better to have this in a separate PR since it's not related to groups handling

Reply:
i think because there were minor changes made to the tests as well, i had to tweak this just a bit. I'll keep this in mind for future PRs